mifosng-android/build.gradle

```diff
@@ -114,6 +115,7 @@ dependencies {
     compile 'com.crashlytics.android:crashlytics:1.+'
     compile 'com.squareup.okhttp:okhttp-urlconnection:2.0.0'
     compile 'com.squareup.okhttp:okhttp:2.0.0'
+    compile 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
```

Comment:
Good Design Thinking and use of the library. However I'd like to point out here that Android has a 65K method limit in older versions and if you compile more than 65K methods you will have to use multi-dexing which is quite inefficient on lower end phones.